tests/testLinearizeLookupTable.py

```diff
@@ -203,7 +203,7 @@ def makeDetector(self, bbox=None, numAmps=None, rowInds=None, colIndOffsets=None
                 ampInfo.setName("amp %d_%d" % (i + 1, j + 1))
                 ampInfo.setBBox(boxArr[i, j])
                 ampInfo.setLinearityType(linearityType)
-                ampInfo.setLinearityCoeffs([rowInds[i, j], colIndOffsets[i, j], 0, 0])
+                ampInfo.setLinearityCoeffs(np.array([rowInds[i, j], colIndOffsets[i, j], 0, 0]))
```

Comment:
I suggest that you specify `dtype=float` for clarity. Otherwise it is not clear why `np.array` is used.